tasks/files-to-granules/package.json

```diff
@@ -36,13 +36,13 @@
   "author": "Cumulus Authors",
   "license": "Apache-2.0",
   "dependencies": {
-    "@cumulus/aws-client": "20.3.0",
+    "@cumulus/aws-client": "21.0.0",
     "@cumulus/cumulus-message-adapter-js": "2.3.0",
     "@cumulus/errors": "20.2.0",
```

Comment:
20.2.0?

Reply:
Oh shoot. These weren't updated in the last release for 20.3? 

https://github.com/nasa/cumulus/pull/4063/commits/af739d1e4c82e9c5b7fb8e4e2bf9ff61cc6e40b8